fix grammatical issues in other_form_controls (#22319)
mdn/content: files/en-us/learn/forms/other_form_controls/index.md

## Drop-down controls

> Drop-down controls are a simple way to let users select from many options without taking up much space in the user interface. HTML has two forms of drop-down content: the **select box**, and the **autocomplete box**. In both cases the interaction is the same — once the control is activated, the browser displays a list of values the user can select between.

Reviewer: ```suggestion
Drop-down controls are a simple way to let users select from many options without taking up much space in the user interface. HTML has two types of drop-down controls: the **select box** and the **autocomplete box**. The interaction is the same in both the types of drop-down controls — after the control is activated, the browser displays a list of values the user can select from.
```

A meter bar represents a fixed value in a range delimited by {{htmlattrxref("max","meter")}} and {{htmlattrxref("min","meter")}} values. This value is visually rendered as a bar, and to know how this bar looks, we compare the value to some other set values:

> - The {{htmlattrxref("low","meter")}} and {{htmlattrxref("high","meter")}} values divide the range into three parts:

Reviewer: ```suggestion
- The {{htmlattrxref("low","meter")}} and {{htmlattrxref("high","meter")}} values divide the range into the following three parts:
```

- If the current value is in the average part of the range, the bar is yellow.
- If the current value is in the worst part of the range, the bar is red.

> Such a bar is created using a {{HTMLElement("meter")}} element. This is for implementing any kind of meter, for example a bar showing the total space used on a disk, which turns red when it starts to get full.

Reviewer: ```suggestion
Such a bar is created by using the {{HTMLElement("meter")}} element. This is for implementing any kind of meter; for example, a bar showing the total space used on a disk, which turns red when it starts to get full.
```
